py3status/modules/prometheus.py

```diff
@@ -94,7 +88,12 @@ def prometheus(self):
         return ret
 
     def _query(self, query):
-        r = self.py3.request(self.server + "/api/v1/query", params={"query": query})
+        try:
+            r = self.py3.request(self.server + "/api/v1/query", params={"query": query})
+        except Exception as e:
```

Comment:
Just like @lasers I would rather display the error on the bar than silencing it please

on_error is the way to go for users who wish to dismiss errors like you